sdk/cdn/azure-mgmt-cdn/azure/mgmt/cdn/models/_models_py3.py

```diff
@@ -3392,7 +3379,7 @@ class RequestHeaderMatchConditionParameters(Model):
 
     odatatype = "#Microsoft.Azure.Cdn.Models.DeliveryRuleRequestHeaderConditionParameters"
 
-    def __init__(self, *, selector: str, operator, match_values, negate_condition: bool=None, transforms=None, **kwargs) -> None:
+    def __init__(self, *, operator, selector: str=None, negate_condition: bool=None, match_values=None, transforms=None, **kwargs) -> None:
```

Comment:
Unfortunately this looks like a breaking change. In the C# SDK, we're customizing the generated code to avoid a breaking change here, is that possible with the python SDK? If not let's make sure this is released as a new major version.

Reply:
@lsmith130 
![image](https://user-images.githubusercontent.com/70930885/100196525-efec2400-2f33-11eb-9d6c-8f9b9a389dc8.png)
picture above indicates that the change makes the parameters optional. Is there anything wrong?

Reply:
I see, these are keyword-only arguments. In that case, please ignore my earlier comment. This looks good to me.